docs/epiviz/inputs.rst

```diff
@@ -8,6 +8,7 @@ EpiViz-AT Inputs
    :maxdepth: 2
 
    bundle
+   priors_as_data
```

Comment:
Should we add a note for "one covariate"?  Perhaps there is a better place for this logic to be  documented.   Basically... The “one covariate”, which is a study covariate of all ones, with covariate_id=1604... I saw you want the one covariate to be a global covariate in EpiViz-AT.  Where would we put that?

Reply:
Under "study and country covariates". Added.